meinberlin/apps/budgeting/assets/Pagination.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import django from 'django'
+
+export const Pagination = (props) => {
+  const {
+    pageSize,
+    count,
+    currPageIndex
+  } = props
+  // FIXME: this is beyond any readability. It just calculates how many
```

Comment:
There should be a way to get all this from the API?!

Reply:
will try if i can provide this via api, if not i will leave for now.

Reply:
I think, it should be count, next and previous. https://www.django-rest-framework.org/api-guide/pagination/#pagenumberpagination